sycl/include/CL/sycl/detail/property_helper.hpp

```diff
@@ -32,7 +32,8 @@ enum DataLessPropKind {
   BufferUsePinnedHostMemory = 5,
   UsePrimaryContext = 6,
   InitializeToIdentity = 7,
-  DataLessPropKindSize = 8
+  // Exceeding 32 may cause ABI breaking change on some of OSes.
+  DataLessPropKindSize = 32
```

Comment:
Suggest defining a new value for max size and use it as a size of bitset. Because this value is also used to check if requested property is "in range of known" in property_list_base.hpp:65.

Reply:
Added in https://github.com/intel/llvm/pull/3458/commits/84f168fe08357a69befea29cec224c255957bbb2

Reply:
Thanks!